PryntTrimmerView/Classes/Parents/AssetVideoScrollView.swift

```diff
@@ -242,12 +215,31 @@ extension AssetVideoScrollView: UICollectionViewDataSource {
             }
         }
 
+        if isFirst(indexPath) {
+            cell.imageView.layer.cornerRadius = 8.0
+            cell.imageView.clipsToBounds = true
+            if #available(iOS 11.0, *) {
+                cell.imageView.layer.maskedCorners = [.layerMinXMinYCorner, .layerMinXMaxYCorner]
+            }
+
+        } else if isLast(indexPath) {
+            cell.imageView.layer.cornerRadius = 8.0
+            cell.imageView.clipsToBounds = true
+            if #available(iOS 11.0, *) {
+                cell.imageView.layer.maskedCorners = [.layerMaxXMinYCorner, .layerMaxXMaxYCorner]
+            }
+        }
```

Comment:
these 2 blocks looks exactly the same. Why not

```suggestion
        if isFirst(indexPath) || isLast(indexPath) {
            cell.imageView.layer.cornerRadius = 8.0
            cell.imageView.clipsToBounds = true
            if #available(iOS 11.0, *) {
                cell.imageView.layer.maskedCorners = [.layerMinXMinYCorner, .layerMinXMaxYCorner]
            }
       }
```

Reply:
In addition, consider adding a function to the cell to increase both single responsibility and readability:

```suggestion
        if isFirst(indexPath) || isLast(indexPath) {
          cell.roundCorners() // or something similar
        }
```

Reply:
they aren't the same but ill add a function depending on which side it is